src/backend/distributed/transaction/multi_shard_transaction.c

```diff
@@ -176,8 +176,12 @@ BeginTransactionOnShardPlacements(uint64 shardId, char *userName)
 		 */
 		MarkRemoteTransactionCritical(connection);
 
-		/* issue BEGIN */
-		RemoteTransactionBegin(connection);
+		/* the special BARE mode (for e.g. VACUUM/ANALYZE) skips BEGIN */
```

Comment:
Well, that's what the _e.g._ was meant for, and anything longer wouldn't fit on one line. Leaving as-is.